### ISSUE TITLE
WDAC is not parsing Treat Revoked/Expired correctly

### ISSUE THREAD

**Issue by jgeurten** [MicrosoftDocs/WDAC-Toolkit#68]
Thanks for reporting this. I will file a bug on myself to investigate tomorrow. I think the issue is I have an older code integrity schema that is being serialized and I need to update to the latest schema. So when the Wizard parses the xml to set the UI, it does not understand "Treat Revoked as Unsigned" and sets the default state in the UI (disabled). 

Will update you by next week!

Thanks,
Jordan

 
Hi Jordan,

 

Small finding re editing policies in the wizard (1.6.5.2.22):

created policy with “Treat Revoked as Unsigned” enabled
verified setting is in the XML
loaded the policy into the wizard
Treat Revoked As Unsigned is no longer enabled in the UI?
 

Curiously, if I then next-next-finish (leaving TraU disabled), the setting *is* again in the XML, but at the top of the list of settings?

In other words: once the setting is in an XML, there is no way of getting rid of it, at least not in the Wizard.
